version/info.go

```diff
@@ -41,14 +42,15 @@ func NewCollector(program string) prometheus.Collector {
 			Namespace: program,
 			Name:      "build_info",
 			Help: fmt.Sprintf(
-				"A metric with a constant '1' value labeled by version, revision, branch, and goversion from which %s was built.",
+				"A metric with a constant '1' value labeled by version, revision, branch, goversion and platform from which %s was built.",
```

Comment:
The "from which %s was built" part is now not fully correct anymore, since the platform it was built on is not always the platform it was built for (and is running on). How about keeping the original string but then ending it as "[...] goversion from which %s was built, and the target platform for the build."